src/components/projects/new/competence-picker.svelte

```diff
@@ -0,0 +1,39 @@
+<script>
+  import { createEventDispatcher } from 'svelte';
+  import Dropdown from 'ui/dropdown.svelte';
+  import CheckboxGroup from 'ui/checkbox-group.svelte';
+
+  export let competences;
+  export let activityIndex;
+  export let value;
+  export let errorNotSpecified;
+
+  $: competenceOptions = competences.map(competence => ({
+    value: competence.id,
+    checked: value.includes(competence.id),
+  }));
+  $: competenceLabels = competences.map(competence => competence.name);
+
+  function processCheckboxChange({ detail: competence }) {
+    if (competence.checked) {
+      value.push(competence.value);
+      value = value;
+    } else {
+      value = value.filter(id => id !== competence.value);
+    }
+    errorNotSpecified = value.length === 0;
+    dispatch('change', value);
+  }
+
+  const dispatch = createEventDispatcher();
+</script>
+
+<Dropdown label="select competences" dropdownclass="btn-shift" noclose nowrap>
+  <CheckboxGroup
+    name="competences{activityIndex}"
+    maxChecked={3}
```

Comment:
I suggest parametrizing this value to allow for reusability when it comes to giving feedback.
Not necessarily right now, though. Just remember it when it comes to the other feature

Reply:
Don't get what you mean

Reply:
When volunteers submit feedback, they are allowed to choose more than 3 competences. My suggestion is to move this number into a prop so that the same component can be used when submitting feedback (if it would be useful there). If the interface itself is different then nvm

Reply:
It is different. In the other picker is a set of chips